Solutions/GoogleCloudPlatformDNS/Hunting Queries/GCPDNSRareErrors.yaml

```diff
@@ -0,0 +1,29 @@
+id: 9e3145ef-f6bf-4dd0-8783-a0847f68cb54
+name: Google DNS - Domains with rare errors
+description: |
+  'Query searches for Domains with rare errors.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: GCPDNSDataConnector
+    dataTypes:
+      - imDNSGCPCloudDNS
+tactics:
+  - CommandAndControl
+relevantTechniques:
+  - T1095
+query: |
+  imDNSGCPCloudDNS
+  | where TimeGenerated > ago(24h)
+  | where EventResultDetails != 'NOERROR'
+  | summarize count() by EventResultDetail, Query
+  | order by count_ asc

```

Comment:
How are we determining which are rare. Wondering if we wanted to use
top 5 or top 10 here.

Reply:
fixed